build/npm/postinstall.ts

```diff
@@ -152,6 +166,7 @@ for (const dir of dirs) {
 
 	if (/^(.build\/distro\/npm\/)?remote$/.test(dir)) {
 		// node modules used by vscode server
+		removeNodePtyPrebuild(dir);
```

Comment:
Adding here would be a noop since we haven't installed deps for this directory, remove and add it in https://github.com/microsoft/vscode/blob/f1db08ab87ec1108d0e3c2a20c7ffe5b8a55a23b/build/npm/postinstall.ts#L71 for other directories